Creating a new element is broken under 2.0.0-next

<!-- Hello! Please read the [Contributing Guidelines](CONTRIBUTING.md) before submitting an issue. -->

## Description of the issue

After updating to the 2.0.0-next, attempting to create a new element results in error of "Cannot find module 'ora'"

<!-- A clear and concise description of what the bug is. -->



### Impacted component(s)

- tools/create-element

### Steps to reproduce


```
npm run clean:deep
npm ci
npm run build --workspace @patternfly/create-element
npm run new
```

The following error occurs: `Error: Cannot find module 'ora'`

```
> new
> ./tools/create-element/bin/main.js

node:internal/modules/cjs/loader:933
  const err = new Error(message);
              ^

Error: Cannot find module 'ora'
Require stack:
- /Users/sps/Sites/_pattern-fly/pfe2/tools/create-element/node_modules/inquirer/lib/utils/screen-manager.js
- /Users/sps/Sites/_pattern-fly/pfe2/tools/create-element/node_modules/inquirer/lib/prompts/base.js
- /Users/sps/Sites/_pattern-fly/pfe2/tools/create-element/node_modules/inquirer/lib/prompts/list.js
- /Users/sps/Sites/_pattern-fly/pfe2/tools/create-element/node_modules/inquirer/lib/inquirer.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/sps/Sites/_pattern-fly/pfe2/tools/create-element/node_modules/inquirer/lib/utils/screen-manager.js:10:13)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/sps/Sites/_pattern-fly/pfe2/tools/create-element/node_modules/inquirer/lib/utils/screen-manager.js',
    '/Users/sps/Sites/_pattern-fly/pfe2/tools/create-element/node_modules/inquirer/lib/prompts/base.js',
    '/Users/sps/Sites/_pattern-fly/pfe2/tools/create-element/node_modules/inquirer/lib/prompts/list.js',
    '/Users/sps/Sites/_pattern-fly/pfe2/tools/create-element/node_modules/inquirer/lib/inquirer.js'
  ]
}
```


<!-- 
  Please update the labels for this component to reflect the topic of the issue: accessibility, doc / demo, functionality, integration, styles-only, tests, tools.
  
  Note also the severity level; all new issues default to severity level 1 which is low priority.  If you feel this issue deserves more attention, please set the label to sev-2 or sev-3.
-->

--- a/.changeset/good-actors-allow.md
+++ b/.changeset/good-actors-allow.md
@@ -1,0 +1,5 @@
+---
+"@patternfly/create-element": patch
+---
+
+Fix 'cannot find module' errors

--- a/.changeset/loud-cycles-vanish.md
+++ b/.changeset/loud-cycles-vanish.md
@@ -1,0 +1,5 @@
+---
+"@patternfly/pfe-tools": patch
+---
+
+Ship built files instead of sources

--- a/package.json
+++ b/package.json
@@ -69,6 +69,7 @@
     "@patternfly/pfe-tools": "^0.0.1",
     "@types/numeral": "^2.0.2",
     "@types/prismjs": "^1.26.0",
+    "@types/prompts": "^2.0.14",
     "commitlint": "^16.1.0",
     "cpx": "^1.5.0",
     "custom-elements-manifest": "^1.0.0",
@@ -78,6 +79,7 @@
     "lerna": "^4.0.0",
     "npm-merge-driver": "^2.3.6",
     "open": "^8.0.4",
+    "prompts": "^2.4.2",
     "renamer": "^4.0.0",
     "sassdoc": "^2.7.3"
   },

--- a/scripts/open-pr.js
+++ b/scripts/open-pr.js
@@ -1,9 +1,10 @@
 /* eslint-env node */
 import branch from 'git-branch';
-import inquirer from 'inquirer';
+import prompts from 'prompts';
 import fs from 'fs';
 import path from 'path';
 import open from 'open';
+import { URL } from 'url';
 
 // Capture the available PR templates
 const getTemplates = loc =>
@@ -36,39 +37,42 @@ const getLabels = filename => {
   return [];
 };
 
-inquirer
-  .prompt([
-    {
-      type: 'list',
-      name: 'template',
-      message: 'Choose your template:',
-      choices: prTemplates,
-      filter: response => `${response.replace(' ', '_')}.md`
-    },
-    {
-      type: 'confirm',
-      name: 'advanced_questions',
-      message: 'Additional information?'
-    },
-    {
-      type: 'input',
-      name: 'base_branch',
-      message: 'Base branch:',
-      default: 'main',
-      when: answers => answers.advanced_questions
-    },
-    {
-      type: 'input',
-      name: 'pr_branch',
-      message: 'PR branch:',
-      default: branch.sync(),
-      when: answers => answers.advanced_questions
-    }
-  ])
-  .then(answers => {
-    // Get the labels from the template
-    const labels = getLabels(`./.github/PULL_REQUEST_TEMPLATE/${answers.template}`) || [];
+const answers = await prompts([
+  {
+    type: 'list',
+    name: 'template',
+    message: 'Choose your template:',
+    choices: prTemplates,
+    filter: response => `${response.replace(' ', '_')}.md`
+  },
+  {
+    type: 'confirm',
+    name: 'advancedQuestions',
+    message: 'Additional information?'
+  },
+  {
+    type: (_, { advancedQuestions }) => !!advancedQuestions && 'input',
+    name: 'baseBranch',
+    message: 'Base branch:',
+    default: 'main',
+  },
+  {
+    type: (_, { advancedQuestions }) => !!advancedQuestions && 'input',
+    name: 'prBranch',
+    message: 'PR branch:',
+    default: branch.sync(),
+  }
+]);
 
-    open(`https://github.com/patternfly/patternfly-elements/compare/${answers.base_branch ||
-    'main'}...${answers.pr_branch || branch.sync()}?template=${answers.template}&labels=${encodeURIComponent(labels.join(','))}`);
-  });
+// Get the labels from the template
+const labels = getLabels(`./.github/PULL_REQUEST_TEMPLATE/${answers.template}`) || [];
+
+const url = new URL(
+  `/patternfly/patternfly-elements/compare/${answers.baseBranch || 'main'}...${answers.prBranch || branch.sync()}`,
+  'https://github.com'
+);
+
+url.searchParams.set('template', answers.template);
+url.searchParams.set('labels', labels.join(','));
+
+open(url.toString());

--- a/tools/create-element/generator/element.ts
+++ b/tools/create-element/generator/element.ts
@@ -3,8 +3,8 @@ import type { CompilerOptions, ProjectReference } from 'typescript';
 
 import Case from 'case';
 import Chalk from 'chalk';
-import inquirer from 'inquirer';
-import execa from 'execa';
+import prompts from 'prompts';
+import { execa, execaCommand } from 'execa';
 
 import { fileURLToPath } from 'url';
 import { dirname, join, relative } from 'path';
@@ -22,13 +22,13 @@ const { green, greenBright } = Chalk;
 const __dirname = dirname(fileURLToPath(import.meta.url));
 
 /**
- * Available filenames.
- *
- * To add a new file to the element template,
- * 1. Add a key to this enum
- * 2. Add the template file's path to `TEMPLATE_FILE_PATHS`
- * 3. Add the output path to `getFilePathsRelativeToPackageDir`, interpolating as needed.
- */
+* Available filenames.
+*
+* To add a new file to the element template,
+* 1. Add a key to this enum
+* 2. Add the template file's path to `TEMPLATE_FILE_PATHS`
+* 3. Add the output path to `getFilePathsRelativeToPackageDir`, interpolating as needed.
+*/
 enum FileKey {
   cemConfig = 'cemConfig',
   component = 'component',
@@ -141,13 +141,16 @@ const getOutputFilePath =
 async function shouldWriteToDir(options: GenerateElementOptions): Promise<boolean> {
   if (options.overwrite || !await exists(getComponentAbsPath(options))) {
     return true;
+  } else if (!options?.directory) {
+    return false;
   } else {
-    return await inquirer.prompt([{
+    const { overwrite = false } = await prompts([{
       type: 'confirm',
       name: 'overwrite',
-      default: false,
+      initial: options.overwrite,
       message: `Directory ${getComponentPathFromDirectoryOption(options)} exists. Overwrite?`,
-    }]).then(({ overwrite = false }) => overwrite);
+    }]);
+    return overwrite;
   }
 }
 
@@ -224,16 +227,17 @@ async function updateTsconfig(options: GenerateElementOptions): Promise<void> {
     config.references.push({ 'path': `./elements/${tagName}` });
   }
   await writeFile(configPath, JSON.stringify(config, null, 2), 'utf8');
-  await execa.command(`npx eslint --fix ${configPath}`);
+  await execaCommand(`npx eslint --fix ${configPath}`);
 }
 
 /**
  * Generate an Element
  */
 export async function generateElement(options: GenerateElementOptions): Promise<void> {
-  if (!options) {
+  // ctrl-c
+  if (!options || !options.tagName) {
     return;
-  } // ctrl-c
+  }
 
   // Quit if trying to scaffold an element in an uninitialized non-monorepo
   if (!await exists(join(options.directory, 'package.json'))) {

--- a/tools/create-element/main.ts
+++ b/tools/create-element/main.ts
@@ -2,7 +2,7 @@ import { generateElement } from './generator/element.js';
 
 import Chalk from 'chalk';
 import Yargs from 'yargs';
-import inquirer from 'inquirer';
+import prompts from 'prompts';
 
 export interface BaseOptions {
   silent: boolean;
@@ -65,16 +65,18 @@ export async function promptForElementGeneratorOptions(
 
   return {
     ...options,
-    ...await inquirer.prompt([{
-      type: 'input',
+    ...await prompts([{
+      type: () => !options?.tagName && 'text',
       name: 'tagName',
       message: 'What is the element\'s tag name?',
+      initial: options?.tagName ?? '',
       validate: name => name.includes('-') || ERR_BAD_CE_TAG_NAME,
     }, {
-      type: 'input',
+      type: () => !options?.scope && 'text',
       name: 'scope',
       message: 'What is the package\'s NPM scope?',
-    }], options),
+      initial: options?.scope ?? ''
+    }]),
   } as GenerateElementOptions;
 }
 

--- a/tools/create-element/package.json
+++ b/tools/create-element/package.json
@@ -49,9 +49,9 @@
   "dependencies": {
     "case": "^1.6.3",
     "chalk": "^4.1.2",
-    "execa": "^5.1.1",
-    "inquirer": "^8.2.0",
+    "execa": "^6.1.0",
     "ncp": "^2.0.0",
+    "prompts": "^2.4.2",
     "tslib": "^2.3.1",
     "yargs": "^17.2.1"
   },

--- a/tools/create-element/templates/element/package.json
+++ b/tools/create-element/templates/element/package.json
@@ -50,7 +50,7 @@
     "web components"
   ],
   "dependencies": {
-    "@patternfly/pfe-core": "^2.0.0",
+    "@patternfly/pfe-core": "^2.0.0 || ^2.0.0-next.0",
     "lit": "^2.0.0"
   }
 }

--- a/tools/pfe-tools/README.md
+++ b/tools/pfe-tools/README.md
@@ -1,0 +1,21 @@
+# `@patternfly/pfe-tools`
+
+Tools and utilities for building PatternFly Elements and other design systems.
+
+## 11ty Helpers
+
+Helpers for collating and rendering
+[custom-elements manifests](https://github.com/webcomponents/custom-elements-manifest/)
+in 11ty sites.
+
+## custom-elements-manifest
+
+Plugins and helpers for working with custom-elements-manifest analyzer
+
+## esbuild-plugins
+
+Helpers for working with esbuild in design system monorepos
+
+## test
+
+Helpers for testing web components using [web test runner](https://modern-web.dev/docs/test-runner/overview/)

--- a/tools/pfe-tools/package.json
+++ b/tools/pfe-tools/package.json
@@ -42,6 +42,16 @@
   "bugs": {
     "url": "https://github.com/patternfly/patternfly-elements/issues"
   },
+  "files": [
+    "!*.ts",
+    "custom-elements.json",
+    "**/*.LEGAL.txt",
+    "**/*.css",
+    "**/*.md",
+    "**/*.d.ts",
+    "**/*.js",
+    "**/*.js.map"
+  ],
   "scripts": {
     "build": "tsc -b .",
     "rebuild": "rm tsconfig.tsbuildinfo && tsc -b .",
@@ -61,7 +71,6 @@
     "@playwright/test": "^1.18.1",
     "@types/estree": "^0.0.50",
     "@types/glob": "7.2.0",
-    "@types/inquirer": "^8.2.0",
     "@types/mkdirp": "^1.0.2",
     "@types/mocha": "^9.1.0",
     "@types/nunjucks": "^3.2.1",

